src/ui/components/Breethe/component.ts

```diff
@@ -97,12 +103,11 @@ export default class Breethe extends Component {
     if (!this.loadedLocal) {
       let transform = await this.local.pull((q) => q.findRecords());
       await this.store.sync(transform);
-      await this.coordinator.activate();
```

Comment:
Is this not necessary? According to http://orbitjs.com/v0.15/guide/getting-started.html#Introducing-a-coordinator it should be? Anyway, I think the change is unrelated to the purpose of this PR.

Reply:
Right, it's not related to this PR. I can remove it, but I found out it's redundant with https://github.com/simplabs/breethe-client/blob/type-annotations/src/utils/data/setup-store.ts#L68. The coordinator is already activated by the time it reaches this line.